src/stores/OverlayStore.ts

```diff
@@ -96,6 +96,7 @@ export class OverlayGlobalSettings {
         astString.add("Labelling", this.labelType);
         astString.add("Color", AstColorsIndex.GLOBAL);
         astString.add("Tol", toFixed(this.tolerance / 100, 2), this.tolerance >= 0.001); // convert to fraction
+        astString.add("System", this.explicitSystem);
```

Comment:
does this work ok in situations where `this.explicitSystem` is undefined?

Reply:
Yes, `astString.add()` checks if the value is undefined